locale/de_DE.UTF-8/LC_MESSAGES/palma.po

```diff
@@ -206,6 +206,17 @@ msgid ""
 "            onclick=\"window.open(this.href); return false;\">MacBook</a>\n"
 "            (RealVNC)."
 msgstr ""
+"Zur Bildschirmfreigabe von Ihrem Laptop benutzt PalMA die VNC Software.\n"
+"            Diese muss einmalig heruntergeladen werden.\n"
+"            Für <a href=\"http://www.bib.uni-mannheim.de/fileadmin/pdf/ub/"
```

Comment:
Yeah, well I would suggest to search and replace all of this links in all languages. I created an issue for that https://github.com/UB-Mannheim/PalMA/issues/41.